src/components/ChatList/elements/ChannelImage.tsx

```diff
@@ -91,7 +90,16 @@ const Big: React.FC<ChannelImageMainProps> = ({type, image, style}) => {
     );
   }
 
-  return null;
+  return (
+    <View
+      style={[styles.containerImage, styles.containerDefaultImage, styles.containersigned_primary]}>
+      <FastImage
+        source={CommunityIcon}
+        resizeMode={FastImage.resizeMode.contain}
+        style={styles.imageDefaultCommunity}
+      />
+    </View>
+  );
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A default return case has been added to handle scenarios where the type does not match any of the predefined cases. This is a good practice as it prevents potential errors when none of the conditions are met.
```diff
- return null;
+ return (
+   <View
+     style={[styles.containerImage, styles.containerDefaultImage, styles.containersigned_primary]}>
+     <FastImage
+       source={CommunityIcon}
+       resizeMode={FastImage.resizeMode.contain}
+       style={styles.imageDefaultCommunity}
+     />
+   </View>
+ );
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->